packages/pluggableWidgets/accordion-web/src/components/Accordion.tsx

```diff
@@ -87,6 +87,10 @@ function AccordionGroupWrapper(props: AccordionGroupWrapperProps): ReactElement
                     )
                 );
 
+                if (headerButtons.length < 0) {
```

Comment:
This will never be true, right? I should be either `=== 0` or `<=`

Reply:
You are completely right. I fixed it 😄  I will rebase in the end.